docs/install.rst

```diff
@@ -413,16 +413,16 @@ Installing pre-built Development Versions of ``astropy``
 Most nights a development snapshot of ``astropy`` will be compiled.
 This is useful if you want to test against a development version of astropy but
 do not want to have to build it yourselves. You can see the
-`available astropy dev snapshots page <https://dev.azure.com/astropy-project/astropy/_packaging?_a=package&feed=nightly&package=astropy&protocolType=PyPI&view=versions>`_
+`available astropy dev snapshots page <https://anaconda.org/astropy/astropy/files?type=pypi>`_
 to find out what is currently being offered.
 
 Installing these "nightlies" of ``astropy`` can be achieved by using ``pip``::
 
-  $ pip install --extra-index-url=https://pkgs.dev.azure.com/astropy-project/astropy/_packaging/nightly/pypi/simple/ --pre astropy
+  $ pip install -i https://pypi.anaconda.org/astropy/simple astropy --pre
```

Comment:
Do we want to also recommend a few more flags like this?

https://github.com/astropy/astropy/blob/3890fd1b770369e67e33e356f35678a7451cc6fc/tox.ini#L113

Reply:
I don't think we need those in the instructions? In particular ``--only-binary :all: `` won't be useful as there are only wheels anyway?

Reply:
`--pre` will also install pre-release versions of any dependencies if they happen to be the latest available on PyPI. I think `--pre` should be fine here as the astropy channel only has astropy, but in sunpy we also use the scipy-wheels-nightly channel so we specify dev dependencies as `astropy>=0.0.dev0` or `matplotlib>=0.0.dev0` etc so we only get the dev version for specific packages and not their dependencies. https://github.com/sunpy/sunpy/blob/main/tox.ini